pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -100,15 +100,16 @@ var _ = Describe("AccessCredentials", func() {
 	})
 
 	It("should handle dynamically updating user/password with qemu agent", func() {
-
 		domName := "some-domain"
 		password := "1234"
 		user := "myuser"
-		base64Str := base64.StdEncoding.EncodeToString([]byte(password))
-		cmdSetPassword := fmt.Sprintf(`{"execute":"guest-set-user-password", "arguments": {"username":"%s", "password": "%s", "crypted": false }}`, user, base64Str)
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).Return("", nil)
+		encodedPassword := base64.StdEncoding.EncodeToString([]byte(password))
+
+		mockConn.EXPECT().LookupDomainByName(gomock.Any()).Return(mockDomain, nil).AnyTimes()
+		mockDomain.EXPECT().Free()
+		mockDomain.EXPECT().SetUserPassword(gomock.Any(), gomock.Any(), gomock.Any())
```

Comment:
Please remind me, does it mean these are expected exactly one time if no quantity is provided?

Reply:
Let's just add `Times(1)`.